### ISSUE TITLE
Feature request: Allow multiple "edit in another tab" nodes that update separately

### ISSUE THREAD

**Issue by JPS-GER** [Lerc/canvas_tab#11]
At the moment, if we have multiple "edit in another tab" nodes, all of them get updated with the same image, if we change something in the canvas editor tab. 

It would be much more flexible if each node would send some unique identifier when clicking on "edit", so the canvas tab can only update/change that specific "edit in another tab" node.

For example I would need up to 5 different masks in my workflow, but at the moment all of them get updated at once.

**Comment by DutchEllie** [comment]
Yeah absolutely, this would be absolutely invaluable for inpainting and other workflows that require multiple different masks

**Comment by Lerc** [comment]
Now if the title of the node and the title of the image are the same, updates to that image go directly to the matching node

https://github.com/Lerc/canvas_tab/assets/139390/4285f866-c2a5-404b-8407-45d1f3167593
